test/atc_test.coffee

```diff
@@ -173,9 +173,15 @@ describe 'hubot-atc', ->
           beforeEach ->
             room.user.say "akatz", "hubot release hubot to staging"
 
-          it "tells you no", ->
-            room.user.say "duncan", "hubot release hubot to staging"
-            expect(lastMessage(room)).to.match /sorry, akatz is releasing hubot\/master to staging/
+          context "and releasing master", ->
+            it "tells you no", ->
+              room.user.say "duncan", "hubot release hubot to staging"
+              expect(lastMessage(room)).to.match /sorry, akatz is releasing hubot\/master to staging/
+
+          context "and releasing a branch", ->
+            it "tells you no", ->
+              room.user.say "duncan", "hubot release hubot/my_branch to staging"
+              expect(lastMessage(room)).to.match /sorry, akatz is releasing hubot\/master to staging/
```

Comment:
@deanobarnett User `duncan` requests a lock so that `duncan` can deploy `hubot/my_branch` to `staging`.
`hubot` responds that user `akatz` has a lock on `staging` and `akatz` is deploying `hubot/master` (line 174).

Before this fix, `hubot` would have responded that `akatz` is deploying `hubot/my_branch`, which is incorrect.